src/ocamlorg_frontend/pages/about.eml

```diff
@@ -50,243 +50,168 @@ Layout.render
+                            Good support for <strong>immutable programming</strong>, i.e., programming without making
+                            destructive updates to data structures. This is present in traditional functional languages
+                            like Scheme, and it's also found in distributed, big-data frameworks like Hadoop.
                         </li>
                         <li>
-                            <strong>High Levels of Safety: </strong>
-                            OCaml is a safe language, and programs are verified by the compiler before they can be
-                            executed. This prevents many programming errors, like confusing an integer and a pointer, or
-                            accessing a non-existent field in a record. More specifically, this protects the integrity
-                            of the data being manipulated by an OCaml program. Although OCaml is statically
-                            type-checked, it does not require that the types of function parameters, local variables,
-                            and so on be explicitly declared like in languages such as C or Java. Much of the necessary
-                            type information is instead automatically inferred by the compiler.
+                            <strong>Type inference</strong>, so you don’t need to annotate every single variable in your
+                            program with its type. Instead, types are inferred based on how a value is used. Available
+                            in a limited form in C# with implicitly typed local variables, and in C++11 with its auto
+                            keyword.
+                        </li>
+                        <li>
+                            <strong>Algebraic data types</strong> and <strong>pattern matching</strong> to define and
+                            manipulate complex data structures, available in Scala and F#.
                         </li>
                     </ul>
                     <p>
-                        In essence, OCaml offers the best of both worlds in terms of performance and security. It
-                        minimises the risk of errors whilst still packing a big performance punch. It manages to do this
-                        thanks to its solid backing in academia and the tireless efforts of the open-source community
-                        that surrounds it.
-                    </p>
-
-                    <div class="divider flex justify-center space-x-5 py-5 lg:py-10">
-                        <span class="rounded-full bg-gray-200 block"></span>
-                        <span class="rounded-full bg-gray-200 block"></span>
-                        <span class="rounded-full bg-gray-200 block"></span>
-                    </div>
-
-                    <h3 id="history">History</h3>
+                        There is something transformative about having all these features together and able to interact
```

Comment:
This is a nice paragraph, but I think it can be added without subtracting the one that starts with 'in essence'?